cstruct-sexp.opam

```diff
@@ -19,7 +19,7 @@ depends: [
   "ocaml" {>= "4.03.0"}
   "dune" {build & >= "1.0"}
   "sexplib" {< "v0.12"}
-  "cstruct" {=version}
+  "cstruct" {>= "3.6.0"}
```

Comment:
Maybe too high ... But fix CI about `ppx_cstruct`. Without this constraint, OPAM wants to install `cstruct.0.4`.